### DISCUSSION TITLE
Cannot access Zigbee administrative interface

### DISCUSSION THREAD

**Discussion by ceysa75** [ct-Open-Source/ct-Smart-Home#37] [Q&A]
Dear ct Team,

Thanks for your explanations which, however, slightly differ from the way as explained in the magazine. Finally - as a total noob in the linux and node red world the installation was successful. However, for some reason accessing the Zigbee administrative interface (Zigbee2MqttAssistant) is not possible. I tried it with http://IPraspberry:8880. But starting node red with http://IPraspberry:1880 works. Is this related with the configuration.yaml file which actually is not fully configured after a fresh install.

I think, I need the Zigbee administrative interface for basic stuff like simply pairing devices, am I right ? Unfortunately, in your magazine it is not described how to simply pair the CC2531 USB sniffer and a simple zigbee device as e.g. a a power plug (in my case the Innr SP 220).

Best regards...

**Comment by ceysa75** [comment]
Dear ct team,

I am struggling with the pairing process. What I think I have learned now is that pairing should be possible directly within the Node Red interface. As far as I understood I need 'node-red-contrib-zigbee' for that. However, it cannot be installed for whatever reason. While downloading and installing other contribs works without any issues, this one in particular cannot be installed. The respective LOG report reveals a lot of errors...
I really need your support with my general pairing issues.

Regards

Ceysa

**Comment by merlinschumacher** [comment]
The zigbee interface will be changed to the native one provided by zigbee2mqtt. Read more about it here: https://www.zigbee2mqtt.io/information/frontend.html
It will be available on port 1881. You can check out the latest master to test it.

I personally use node-red-contrib-zigbee, and it will be included in the next release. But I never used it to manage the bridge – only for the devices. Try creating a bridge-node, add your bridge, then click "refresh all data" and done. Then reopen the bridge node and you'll see three additional tabs with various settings.

**Reply by ceysa75** [reply]
Dear Merlin,

without going into detail I found my own way to make all the Node-Red stuff work. However, there was no chance with the ct docker approach. I just went the 'regular' way as described on the official Node-Red pages. And I am an happy Node-Red user now. Unfortunately, It took me several days that I realized that it was not me (or my missing linux knowledge) causing the problems. It was all related on the (maybe outdated) ct-smarthome package. A really disappointing experience, at least in my case...
